source/cli-translate.js

```diff
@@ -0,0 +1,128 @@
+var imagemagick_cli_lists = null;
```

Comment:
Can you move this file to the `assets` folder instead?

Reply:
Sure.